airbyte-webapp/src/components/connection/CatalogTree/streamConfigHelpers/streamConfigHelpers.test.ts

```diff
@@ -15,34 +18,54 @@ const mockStreamConfiguration: AirbyteStreamConfiguration = {
   destinationSyncMode: "overwrite",
 };
 
-const FIELD_ONE = ["field_one"];
-const FIELD_TWO = ["field_two"];
-const FIELD_THREE = ["field_three"];
+const FIELD_ONE: SyncSchemaField = {
+  path: ["field_one"],
+  cleanedName: "field_one",
+  key: "field_one",
+  type: "string",
+};
+const FIELD_TWO: SyncSchemaField = {
+  path: ["field_two"],
+  cleanedName: "field_two",
+  key: "field_two",
+  type: "string",
+};
+const FIELD_THREE: SyncSchemaField = {
+  cleanedName: "field_three",
+  type: "todo",
+  key: "field_three",
+  path: ["field_three"],
+};
```

Comment:
I redefined `FIELD_ONE`, `FIELD_TWO` and `FIELD_THREE` in this file, which is one of the reasons for the large diff.